packages/jest-jasmine2/src/jasmine/spy_registry.js

```diff
@@ -129,6 +133,75 @@ export default function SpyRegistry(options: Object) {
     return spiedMethod;
   };
 
+  this._spyOnProperty = function(obj, propertyName, accessType = 'get') {
+    if (!obj) {
+      throw new Error(
+        'spyOn could not find an object to spy upon for ' + propertyName + '',
```

Comment:
There is an empty string at the end. Can we end the sentence with a `.`? Also, can we prefix the error with "Jest: "?

Reply:
i will do it.

Reply:
@cpojer in the end i've used .see https://github.com/facebook/jest/pull/5107/commits/bd44ca5eba27d91d0bbe483f42823ceb4cef740b